kuma/javascript/src/payments/pages/management.jsx

```diff
@@ -0,0 +1,200 @@
+                                {gettext('Amount')}
+                            </span>
+                            <span className="value">
+                                {/* amount is in cents, so divide by 100 to get dollars */}
+                                {formatMoney(locale, subscription.amount / 100)}
+                            </span>
+                        </li>
+                        <li>
+                            <span className="label credit-card">
+                                {gettext('Card number')}
+                            </span>
+                            <span className="value">{`**** **** **** ${subscription.last4}`}</span>
+                        </li>
+                        <li>
+                            <span className="label credit-card">
+                                {gettext('Expiry')}
+                            </span>
+                            <span className="value">
+                                {subscription.expires_at}
+                            </span>
+                        </li>
+                    </ul>
+                </div>
+                <button
+                    className="cta cancel"
+                    onClick={handleClick}
+                    type="button"
+                >
+                    {gettext('Cancel subscription')}
+                </button>
+                {showForm && (
+                    <CancelSubscriptionForm
+                        onSuccess={handleDeleteSuccess}
+                        setShowForm={setShowForm}
+                        date={lastActiveDate}
+                    />
+                )}
+            </>
+        );
+    };
+
+    const renderContent = () => {
+        switch (true) {
+            case !userData.isAuthenticated:
+                // User is not logged in
+                return (
+                    <Interpolated
+                        id={gettext('Please <signInLink /> to continue.')}
+                        signInLink={<SignInLink text={gettext('sign in')} />}
+                    />
+                );
+            case !userData.isSubscriber || canceled:
+                // Not a subscriber or just canceled their subscription
+                return renderInvitation();
+            case userData.isSubscriber && !!subscription:
+                // Is a subscriber and has subscriptions
+                return renderSubscriptions();
+            case status === 'error':
+                // Something went wrong with fetching data
+                return <ErrorMessage />;
+            default:
+                // Fetching data
+                return <strong>{gettext('Loading…')}</strong>;
+        }
+    };
+
+    return (
+        <>
+            <Subheader title={title} />
+            <main
+                className="contributions-page manage-subscriptions"
+                role="main"
+                data-testid="management-page"
+            >
+                <section>
```

Comment:
If there is no styling attached to this `section` element, I would recommend removing it. We do not get any accessibility or semantic benefit.

Reply:
There is styling attached to this that adds padding around the edges. I tried to move it to `<main>` but it doesn't look quite right.

Reply:
Something to look into in future. Not worth spending more time on it now. Thank you Mindy!